include/ignition/gazebo/rendering/SceneManager.hh

```diff
@@ -214,6 +214,14 @@ inline namespace IGNITION_GAZEBO_VERSION_NAMESPACE {
         std::vector<common::TrajectoryInfo>& _trajectories,
         std::chrono::steady_clock::time_point _time);
 
+    /// \brief Load Actor trajectories
+    /// \param[in] _actor Actor
```

Comment:
Can you document all params? Also I believe `_id` doesn't need to be passed as a reference.

Reply:
Done.